docs/development/getting-started-locally.md

```diff
@@ -0,0 +1,120 @@
+# Etcd-Druid Local Setup
+
+This page aims to provide steps on how to setup Etcd-Druid locally with and without storage providers
+
+## Setting up etcd-druid
+
+### Clone the Etcd-Druid Github repo
+
+```sh
+# clone the repo
+git clone https://github.com/gardener/etcd-druid.git
+# cd into etcd-druid folder
+cd etcd-druid
+```
+
+---
+
+> **Note:** To setup Etcd-Druid with backups enabled on a [LocalStack](https://github.com/localstack/localstack) provider, refer [this document](https://github.com/gardener/etcd-druid/blob/master/docs/development/getting-started-locally-localstack.md)
+
+---
+
+### Setting up the Kind Cluster
+
+```sh
+# Create a kind cluster
+make kind-up
+```
+
+This creates a new kind cluster and stores the kubeconfig in the  `./hack/e2e-test/infrastructure/kind/kubeconfig` file.
+
+To target this newly created cluster, set the `KUBECONFIG` environment variable to the kubeconfig file located at `./hack/e2e-test/infrastructure/kind/kubeconfig` by using the following
+
+```sh
+export KUBECONFIG=$PWD/hack/e2e-test/infrastructure/kind/kubeconfig
+```
+
+### Setting up Etcd-Druid
+
+Generates the Etcd CRD and deploy an etcd-druid pod into the cluster
+
+```sh
+make deploy
+```
+
+### Applying the etcd CR
+
+- **Without Backups enabled**
```

Comment:
Can we have two sub-sections here? 
First would be `Prepare the Etcd CR` where we say the following
1. Where to find the Etcd CR
2. Comment out the `spec.backup.store` section 

Second section would be `Apply the Etcd CR` where we have the example `kubectl apply...` command

Reply:
Agreed with @aaronfern 's point, with slight changes. You can choose to have a structure like this:
```
- Prepare the Etcd CR
  Where to find the etcd CR...
  - Without backups enabled:
    - comment out the spec.backup.store section
  - With backups enabled:
    - prepare secret
    - apply secret
    - populate etcd spec.backup.store appropriately
- Apply the Etcd CR
  How to apply the prepared etcd CR...
```

I think this flow seems quite simple, without any repetition of information/steps.